plugin/src/test/java/org/jenkinsci/plugins/structs/describable/DescribableModelTest.java

```diff
@@ -63,7 +63,7 @@ public static void isUnitTest() {
         Main.isUnitTest = true; // suppress HsErrPidList
     }
 
-    private static final Logger logger = Logger.getLogger(DescribableModelTest.class.getName());
+    private static final Logger logger = Logger.getLogger(DescribableModel.class.getPackage().getName());
```

Comment:
Because there is logging in various classes in the same package, not just `DescribableModel` itself. (@kohsuke probably missed that fact when copying this code from `DescribableHelperTest`.)